Kassiopeia/Terminators/Include/KSTermOutput.h

```diff
@@ -42,6 +42,7 @@ template<class XValueType> class KSTermOutput : public KSComponentTemplate<KSTer
         }
 
         if (*fValue >= fMaxValue || *fValue <= fMinValue) {
+            *fValue = 0;
```

Comment:
```suggestion
```

This change is both uncorrelated to the intent of this merge request and also is just a workaround for an underlying configuration issue, as discussed in #100 .